### DISCUSSION TITLE
React Server Component support

### DISCUSSION THREAD

**Discussion by AsuraKev** [emotion-js/emotion#3119] [Q&A]
Hi there, just wondering if there is going to be a support for RSC in the future? thanks

**Comment by wdcryer** [comment]
Based on [this discussion](https://github.com/emotion-js/emotion/discussions/2827#discussioncomment-3446525), it seems unlikely that Emotion will ever support components that are purely server side rendered. It will likely always rely on client side JS.

**Reply by AsuraKev** [reply]
Thanks. Looks like time to go back to css module and tailwind 🙁
